### DISCUSSION TITLE
A "trigger" html attribute like from htmx

### DISCUSSION THREAD

**Discussion by zraly** [jiripudil/Naja#359] [General]
Hello,

I just came across [htmx](https://github.com/bigskysoftware/htmx) and I like some of the ideas there, that simplify working with ajax. Mostly the parameter [`trigger`](https://htmx.org/attributes/hx-trigger/), which might be a awesome addition to Naja:

`< form hx-trigger="keyup changed delay:500ms">`

Would it make any sense to implement it?

**Comment by jiripudil** [comment]
Hi, it's always been Naja's philosophy to only provide the core set of features, along with extension points for each of them, so that it's easy to build upon it. This trigger thing looks intriguing, but it seems like something that can easily be built on top of Naja, and therefore I don't think it would fit within Naja itself.

But I'll be happy to link to third-party packages that extend Naja :)
